docs/user-guide/knowledge-distillation.rst

```diff
@@ -0,0 +1,362 @@
+               data.data.hf_dataset=True \
+               top_k=4 \
+               model.global_batch_size=16 \
+               model.micro_batch_size=2 \
+               start_from_idx=0 \
+               end_at_idx=56439 \
+               output_path=data/oasst/train_with_logits_0.jsonl
+
+
+    .. tab-item:: Slurm
+        :sync: key4
+
+         To generate the teacher logits via slurm, run the following command: 
+
+         ### TODO: make the mounts more explicit
+
+         .. code-block:: bash 
+
+            #!/bin/bash
+            #SBATCH -A <<YOUR ACCOUNT>>
+            #SBATCH -p <<<YOUR PARTITION>>>
+            #SBATCH -N 1
+            #SBATCH -t 2:00:00
+            #SBATCH -J <<<JOB NAME>>>
+            #SBATCH --ntasks-per-node=8
+            #SBATCH --gpus-per-node=8
+            #SBATCH --exclusive
+            #SBATCH --overcommit
+
+            GPFS="/path/to/nemo-aligner-repo"
+
+            PRETRAINED_ACTOR_NEMO_FILE="/path/to/your/mcore_gpt.nemo"
+
+            PROJECT=WANDB_PROJECT # if you want to use wandb
+
+            RESULTS_DIR="/path/to/result_dir"
+
+            OUTFILE="${RESULTS_DIR}/sft-%j_%t.out"
+            ERRFILE="${RESULTS_DIR}/sft-%j_%t.err"
+            mkdir -p ${RESULTS_DIR}
+
+            CONTAINER=<<<CONTAINER>>> # use the latest NeMo Training container, Aligner will work there
+
+            MOUNTS="--container-mounts=MOUNTS" # mounts
+
+            read -r -d '' cmd <<EOF
+            echo "*******STARTING********" \
+            && echo "---------------" \
+            && echo "Starting..." \
+            && cd ${GPFS} \
+            && export PYTHONPATH="${GPFS}:${PYTHONPATH}" \
+            && export HYDRA_FULL_ERROR=1 \
+            && python examples/nlp/synthetic_data_gen/compute_topk_logits.py \
+                  trainer.num_nodes=\${SLURM_JOB_NUM_NODES} \
+                  trainer.devices=\${SLURM_NTASKS_PER_NODE} \
+                  trainer.precision=bf16 \
+                  pretrained_checkpoint.restore_from_path=teacher_checkpoint/Nemotron-3-8B-Chat-4k-SFT.nemo \
+                  model.megatron_amp_O2=True \
+                  model.tensor_model_parallel_size=1 \
+                  data.chat=True \
+                  data.sample=True \
+                  data.num_workers=0 \
+                  data.data.max_seq_length=4096 \
+                  data.data.file_path=data/oasst/train.jsonl \
+                  data.data.add_eos=False \
+                  data.data.hf_dataset=True \
+                  top_k=4 \
+                  model.global_batch_size=16 \
+                  model.micro_batch_size=2 \
+                  start_from_idx=0 \
+                  end_at_idx=56439 \
+                  output_path=data/oasst/train_with_logits_0.jsonl
+            EOF
+
+            srun -o $OUTFILE -e $ERRFILE --container-image=$CONTAINER $MOUNTS bash -c "${cmd}"
+            set +x
+
+
+You can also generate the teacher logits for the validation dataset by replacing these lines
```

Comment:
fix punctuation.

You can also generate the teacher logits for the validation dataset by replacing these lines: